learning_journal/learning_journal/views/default.py

```diff
@@ -0,0 +1,73 @@
+from pyramid.response import Response
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPNotFound
+from pyramid.httpexceptions import HTTPFound
+import datetime
+
+from sqlalchemy.exc import DBAPIError
+
+from ..models import MyEntry
+
+
+@view_config(route_name="home", renderer="../templates/home.jinja2")
+def home_view(request):
+    try:
+        query = request.dbsession.query(MyEntry)
+        entries = query.all()
+    except DBAPIError:
+        return Response(db_err_msg, content_type="text/plain", status=500)
```

Comment:
for a production-ready site you might consider just letting the DBAPIError raise.  If you don't handle it, Pyramid will turn it into a 500 anyway, and the end result will be the same.  Except you won't display a message giving a visitor information about how to initialize your project database.